odl/contrib/param_opt/param_opt.py

```diff
@@ -72,17 +76,15 @@ def func(lam):
     # Pick resolution to fit the one used by the space
     tol = np.finfo(phantoms[0].space.dtype).resolution * 10
 
-    initial_param = np.asarray(initial_param)
-
-    # We use a faster optimizer for the one parameter case
-    if initial_param.size == 1:
-        bracket = [initial_param - tol, initial_param + tol]
+    if not multivariate:
```

Comment:
Great change overall. This gives users the power to pick themselves, a much better way to do it!